src/script/components/MessagesList/MessageList.tsx

```diff
@@ -121,6 +122,8 @@ export const MessagesList: FC<MessagesListParams> = ({
   const [highlightedMessage, setHighlightedMessage] = useState<string | undefined>(conversation.initialMessage()?.id);
   const conversationLastReadTimestamp = useRef(conversation.last_read_timestamp());
 
+  const [rightMarginWidth, setRightMarginWidth] = useState<number>(0);
```

Comment:
This also can be ref, to avoid each time getting recalculated and re-rendered all the time, but should be recalculated on the language switch.